webob/response.py

```diff
@@ -1248,7 +1270,15 @@ def conditional_response_app(self, environ, start_response):
         * Range               (406 Partial Content; only on GET, HEAD)
         """
         req = BaseRequest(environ)
-        headerlist = self._abs_headerlist(environ)
+
+        # Build headerlist, if there is a Location header, make it absolute
+        headerlist = [
```

Comment:
Good idea, but:

I've been thinking about removing this entirely. Making the `Location` absolute really doesn't provide much especially since all clients since HTTP 1.0 support relative Location's. It's doing extra work that feels unnecessary.

Thoughts?